docker-compose.yml

```diff
@@ -41,27 +52,37 @@ services:
       POSTGRES_DB: PracticalDeveloper_development
     ports:
       - "5432:5432"
+    volumes:
+      - db_data:/var/lib/postgresql/data:Z
+
   redis:
     image: "redis"
     ports:
       - "6379:6379"
+
   elasticsearch:
     image: docker.elastic.co/elasticsearch/elasticsearch:7.5.2
     container_name: elasticsearch
     environment:
-      - cluster.name=docker-cluster
+      - cluster.name=devto
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
       - "discovery.type=single-node"
       - xpack.security.enabled=false
       - xpack.monitoring.enabled=false
       - xpack.graph.enabled=false
       - xpack.watcher.enabled=false
+    volumes:
+      - es_data:/usr/share/elasticsearch/data:Z
+    ports:
+      - "9200:9200"
     ulimits:
       memlock:
         soft: -1
         hard: -1
-    volumes:
-        - ./docker_data/elasticsearch/data:/usr/share/elasticsearch/data
-    ports:
-      - "9200:9200"
+
+volumes:
+  db_data:
```

Comment:
We have a `.gitignore` for `docker_data`:

```
# ES docker volume
docker_data/
```

So I guess we either need to update that or keep both `db_data` and `es_data` in there.

Reply:
Resolved with https://github.com/thepracticaldev/dev.to/pull/7747/commits/23375fb71cf8615b15bfc6ce4f446da80ae9f41e